.ruff.toml

```diff
@@ -0,0 +1,17 @@
+select = [
+  "E", # pycodestyle errors
+  "F", # pyflakes errors
+  "ISC", # flake8-implicit-str-concat
+  "PGH", # pygrep-hooks
+  "RUF100", # unused noqa (yesqa)
+  "UP", # pyupgrade
+  "W", # pycodestyle warnings
+  "YTT", # flake8-2020
+]
+extend-ignore = [
+  "E203", # Whitespace before ':'
+  "E221", # Multiple spaces before operator
+  "E226", # Missing whitespace around arithmetic operator
+  "E241", # Multiple spaces after ','
```

Comment:
Short answer: I copied the config from another project :)

Longer answer: If I remember correctly, they're incompatible with the Black formatter/PEP 8. I was going to add Black and isort in a follow-up PR because it's a bigger diff, but these exclusions are not needed in this PR. 👍